samples/src/voiceMain/kotlin/VoiceBot.kt

```diff
@@ -10,37 +10,43 @@ import dev.kord.core.behavior.interaction.respondPublic
 import dev.kord.core.event.interaction.GuildChatInputCommandInteractionCreateEvent
 import dev.kord.core.on
 import dev.kord.voice.AudioFrame
+import kotlinx.coroutines.DelicateCoroutinesApi
+import kotlinx.coroutines.GlobalScope
 import kotlinx.coroutines.launch
-import kotlinx.coroutines.runBlocking
 
-fun main(args: Array<String>) = runBlocking {
-    val kord = Kord(args.firstOrNull() ?: error("Missing token"))
+@OptIn(DelicateCoroutinesApi::class)
+fun main(args: Array<String>) {
```

Comment:
It might be better to have nonNative main function which is suspending and then use runBlocking on native